content/blog/2020-03-16-teaching-online-on-short-notice/index.Rmarkdown

```diff
@@ -0,0 +1,215 @@
+
+1.  *Your voice matters more than your face.*
+    You can switch off video if you have to,
+    but if you lose sound,
+    your class is over.
+    Eight dollars will get you a Yeti or Samson Meteor microphone;
+    another thirty will get you a stand
+    (because if it's on your desk, all your audience will hear is your typing).
+    You can also use a headset with a microphone,
+    but this can become uncomfortable after a couple of hours,
+    particularly if you wear glasses.
+
+1.  *Turn off notifications (theirs too).*
+    It's rude to check your phone or read email during an in-person meeting,
+    so don't do it when you're teaching
+    (not even while your learners are working on an exercise).
+    You probably can't persuade your learners to be similarly respectful,
+    but the subset you're talking to at any particular time might.
+
+1.  *Share work in real time.*
+    Have learners share their work in a Google Doc or [HackMD][hackmd] during the lesson.
+    (This is no more or no less intimidating than asking a music student to play something for the class.)
+    When you do this,
+    paste in a list of people's names so that they know where to type.
+    Once they're comfortable with this (which will take a few hours),
+    have them take shared notes in the same doc so that your copilot can see what they think you're saying.
+
+1.  *Keep track of who has and hasn't spoken.*
+    Again,
+    a laissez faire class an quickly turn into a discussion among a couple of extroverts,
+    so keep a tally of who's spoken how often and try to even it out.
+    You don't need technology to do this:
+    a scrap of paper and a pencil will work just fine.
+
+1.  *Ask—fill—call.*
+    Thinking takes time.
+    So does typing,
+    and there’s always lag,
+    so when asking for questions or suggestions,
+    give people time to respond,
+    but talk while you're waiting for answers in order to avoid dead air.
+
+1.  *Ask for predictions.*
+    Online polls help keep people engaged but need to be set up in advance.
+    Asking people, "What's going to happen when we run this code?"
+    is a good just-in-time alternative.
+
+1.  *Use breakout rooms.*
+    [Peer instruction][peer-instruction] is the most effective scalable teaching practice we know of.
+    It relies on students debating their answers to a motivating question for two or three minutes at a time.
+    [Zoom][zoom]'s breakout rooms make this relatively easy to run online:
+    it takes 15-30 seconds to get everyone into rooms,
+    and in a class of forty there will be one or two who initially have problems,
+    but it does a lot to keep everyone engaged and motivated.
+
+1.  *Take a break every hour*
+    because the basic unit of teaching is the bladder.
+
+The most important thing to remember is that nobody really knows how to do this—not yet—but
+we'll all know a lot more in a few months thanks to all of the unplanned experiments we're about to launch.
```

Comment:
It's not an "experiment". I think this term got criticized recently in this context. All the unplanned "learning experiences"? I'm not sure, but I'd avoid "experiment".

Reply:
Suggestion: "...thanks to all the brave educators who are about to enter this uncharted territory."?

Reply:
taken out completely to make the ending punchier.